packages/react-dev-utils/printHostingInstructions.js

```diff
@@ -39,7 +39,7 @@ function printHostingInstructions(
   console.log();
   console.log('Find out more about deployment here:');
   console.log();
-  console.log(`  ${chalk.yellow('https://create-react-app.dev/docs/deployment')}`);
+  console.log(`  ${chalk.yellow('bit.ly/CRA-deploy')}`);
```

Comment:
I think the placeholders make sense, but I'd keep this as `https://create-react-app.dev/docs/deployment` so that the end users don't have to follow 3 links to end up in the same page.

Reply:
Whatever links we put in there basically have to work forever and I don't want to have a long list of old links to keep track of. We have a new domain name that we are going to switch all shortlinks over to.